tools/integration_tests/util/client/storage_client.go

```diff
@@ -49,6 +52,14 @@ func CreateStorageClient(ctx context.Context) (client *storage.Client, err error
 	if err != nil {
 		return nil, fmt.Errorf("storage.NewClient: %w", err)
 	}
+	client.SetRetry(
```

Comment:
Can we write a comment here?

Can we write a comment here?

// RetryAlways causes all operations to be retried when the service returns transient error, regardless of idempotency considerations. Since the concurrent execution of our CI/CD tests (VMs, threads) doesn't share any cloud-storage resources, hence it's safe to disregard idempotency.

Reply:
Added